DataFormats/METReco/interface/BeamHaloSummary.h

```diff
@@ -70,6 +73,12 @@ namespace reco {
     std::vector<int>& GetGlobaliPhiSuspects() { return GlobaliPhiSuspects ;}
     const std::vector<int>& GetGlobaliPhiSuspects() const { return GlobaliPhiSuspects ;}
 
+    std::vector<std::vector<std::pair<char, CaloTowerDetId> > >& GetProblematicStrips() { return ProblematicStrips ;}
```

Comment:
A ```std::vector< [some type]>``` can be stored much more efficiently by ROOT (both in time and disk space) than a ```std::vector<std::vector<[some other type]>>```.